atom/browser/browser_mac.mm

```diff
@@ -112,12 +112,16 @@
 void Browser::SetAppUserModelID(const base::string16& name) {
 }
 
-void Browser::SetUserActivity(
-    const std::string& type,
-    const base::DictionaryValue& user_info) {
+void Browser::SetUserActivity(const std::string& type,
+                              const base::DictionaryValue& user_info,
+                              mate::Arguments* args) {
+  std::string URL = "";
```

Comment:
Passing string directly may have [escaping problems](https://github.com/electron/electron/issues/2768), the safe way is to use `net::NSURLWithGURL` to convert the url to NSURL.